deps/rabbit/src/rabbit_peer_discovery_classic_config.erl

```diff
@@ -26,7 +26,8 @@ list_nodes() ->
         Nodes when is_list(Nodes)  -> {ok, {Nodes, disc}}
```

Comment:
BTW, I couldn't get the peer_discovery_classic_config_SUITE.erl tests running. They keep timing out.

```
make ct-peer_discovery_classic_config
```

So, I'm hoping that when I create a PR to official RabbitMQ repo, that test suite will be executed.

Alternatively, if you know how to make that test suite runs locally or on our GitHub fork, please let me know!